### PR TITLE
Add a badge for the functional tests

### LINKED ISSUE #883: Add status badge to readme for nightly product testing
# 💡 Summary #

Add a workflow status badge to the ScubaGear readme.

Part of Epic #1209

## Motivation and context ##

This would be useful because it would allow people to see at a glance if the current build of ScubaGear (on the main branch) can successfully test the various Microsoft services.

## Implementation notes ##

GitHub provides the info necessary to implement here:  https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

## Acceptance criteria ##

How do we know when this work is done?

- [ ] There's a badge on the readme that accurately reflects the current status of nightly testing on the main branch.

### PR DESCRIPTION
**Pull request by james-garriss**
## 🗣 Description ##

Now that the functional tests are passing regularly, it's time to proudly display a badge on the front page of the readme.  Actually, I added 2 badges, one for the nightly tests and one for the CI pipeline.

## 💭 Motivation and context ##

Closes #883

## 🧪 Testing ##

Created test repo, added badge, verified that it worked.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] PR targets the correct parent branch (e.g., main or release-name) for merge.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] Changes are sized such that they do not touch excessive number of files.
- [x] These code changes follow the ScubaGear [content style guide]
- [x] Related issues these changes resolve are linked preferably via [closing keywords]
- [x] All relevant type-of-change labels added.
- [x] All relevant project fields are set.
- [x] All relevant repo and/or project documentation updated to reflect these changes.
- [x] All relevant functional tests passed.
- [x] All automated checks (e.g., linting, static analysis, unit/smoke tests) passed.

## ✅ Pre-merge checklist ##

- [x] PR passed smoke test check.
- [x] Feature branch has been rebased against changes from parent branch, as needed
- [x] Resolved all merge conflicts on branch
- [x] Notified merge coordinator that PR is ready for merge via comment mention

## ✅ Post-merge checklist ##

- [ ] Feature branch deleted after merge to clean up repository.
- [ ] Verified that all checks pass on parent branch (e.g., main or release-name) after merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 [![GitHub Release][github-release-img]][release]
 [![PSGallery Release][psgallery-release-img]][psgallery]
-[![GitHub CICD Workflow][github-cicd-workflow-img]][github-cicd-workflow]
+![CI Pipeline](https://github.com/cisagov/ScubaGear/actions/workflows/run_pipeline.yaml/badge.svg)
```
Comment (mitchelbaker-cisa) [inline README.md:5]: Change these to point of reference links for consistency, examples are defined at the bottom of the current README.md or you can look at #494 as well.

Screenshot (210)

**Review by mitchelbaker-cisa — CHANGES_REQUESTED** [review]
One small comment to address but looks awesome otherwise, love to see these passing.

**Review by dagarwal-mitre — APPROVED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (james-garriss) [reply README.md]: Done

**Comment by mitchelbaker-cisa** [comment]
@nanda-katikaneni ready to merge

**Pull request merged**